### ISSUE TITLE
animateToRegion with duration 0 not working on Android 

### ISSUE THREAD

**Issue by MichaelHolley** [react-native-maps/react-native-maps#4338]
### Summary

Calling the animateToRegion-method with a duration of 0 has no effect on android devices.
We use this method on a search-button event. Meaning, we provide a region and want the map to move this location instantly.

This issue has not been present in version <1.0.0

### Reproducible sample code

```TSX
const adjustedRegion = {
            ...this.position,
            latitude: adjustedLatitude,
        };

this.mapView.animateToRegion(adjustedRegion, 0);
```

### Steps to reproduce

Animate to region with a duration of 0 on a button-event for a region which is not equal to the current position.

### Expected result

Jump to the region without delay/animation.

### Actual result

The map does not move at all and stays at its current position.

### React Native Maps Version

1.0.0

### What platforms are you seeing the problem on?

Android

### React Native Version

0.67.1

### What version of Expo are you using?

SDK 44

### Device(s)

Android devices/emulators

**Comment by Simon-TechForm** [comment]
Hello @MichaelHolley 👋. I've just released v1.0.3 that should fix the issue. Could you check it out and report back to me?

**Comment by MichaelHolley** [comment]
> Hello @MichaelHolley 👋. I've just released v1.0.3 that should fix the issue. Could you check it out and report back to me?

Hey there. I have just checked out v1.0.3 and the issue seems to be resolved. 😊 Thank you for your quick solution.

**Comment by ramzzzes** [comment]
its not working on ANDROID because "animateToRegion"  also needs  "delta" params



### Not working : 
```
latitude: 41.012031,
longitude: 39.01001,
```
 

### Working :
```
latitude: 41.012031,
longitude: 39.01001,
latitudeDelta: 0.04,
longitudeDelta: 0.05
```
